README.md

```diff
@@ -1,10 +1,9 @@
-# <p align="center">🧬 Spore Protocol</p>
+# <p align="center"> Spore Protocol</p>
 <p align="center">
   A protocol for valuing on-chain contents, build on top of <a href="https://github.com/nervosnetwork/ckb">CKB</a>. Check <a href="https://docs.spore.pro">Spore Docs</a> for a quick start.
 </p>
 
-
-Spore developers are supposed to use [Spore SDK](https://github.com/sporeprotocol/spore-sdk) instead of this project directly. But if you want to test/extend the contract or deploy on a local chain, you can follow the [Development](#⚙️-development) part.
+Developers are encouraged to utilize the [Spore SDK](https://github.com/sporeprotocol/spore-sdk) for creating applications using Spore, rather than working directly with this contracts. However, if your want to test/extend the contract, or deploy it on a local blockchain, then you can refer to the [Development](#⚙️-development) section.
```

Comment:
```suggestion
Developers are encouraged to utilize the [Spore SDK](https://github.com/sporeprotocol/spore-sdk) for creating applications using Spore, rather than working directly with this contracts. However, if you want to test/extend the contract, or deploy it on a local blockchain, then you can refer to the [Development](#⚙️-development) section.
```